compiler/rustc_parse/src/parser/pat.rs

```diff
@@ -738,6 +771,19 @@ impl<'a> Parser<'a> {
             }
             None
         };
+
+        if let Some(span) = open_paren {
```

Comment:
```suggestion
        if let Some(open_paren) = open_paren {
```

or `open_paren_sp` or sth. like that (as long as names for the opening and the closing parens are consistent >.<).